pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -228,30 +236,34 @@ def on_epoch_end(self, epoch, logs=None):
         self.epochs_since_last_save += 1
         if self.epochs_since_last_save >= self.period:
             self.epochs_since_last_save = 0
-            filepath = '{}/{}_ckpt_epoch_{}.ckpt'.format(self.filepath, self.prefix, epoch + 1)
-            if self.save_best_only:
+            if self.save_all or self.save_last:
+                overwrite = not self.save_all
+                filepath = '{}/{}_ckpt_epoch_{}.ckpt'.format(self.filepath, self.prefix, epoch + 1)
+                self.save_model(filepath, overwrite=overwrite, prefix=self.prefix)
+                if self.verbose > 0:
+                    print('\nEpoch %05d: saving model to %s' % (epoch + 1, filepath))
+            if self.save_best:
                 current = logs.get(self.monitor)
                 if current is None:
                     print('Can save best model only with %s available,'
                           ' skipping.' % (self.monitor), RuntimeWarning)
                 else:
                     if self.monitor_op(current, self.best):
+                        prefix = 'best_{}'.format(self.prefix)
```

Comment:
I would rather keep the same schema - `'{}/{}_ckpt_epoch_{}.ckpt'` where the last element is either 
* epoch number
* `best`
* `last`

Reply:
Will do